packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -356,6 +376,7 @@ class _MenuAnchorState extends State<MenuAnchor> {
     if (!widget.anchorTapClosesMenu) {
```

Comment:
Just a question. In the future when we deprecate `anchorTapClosesMenu `, we can just directly remove this if statement, right?

Reply:
Yes. But maybe I should point out in the docs that it needs to be `false` for `consumeOutsideTap` to be effective.